check_changed.sh

```diff
@@ -41,7 +41,7 @@ if [ -z "$CHANGED_FILES" ]
 then
     # nothing changed, skip building
     # this indicates to the parent script that the build can be terminated
-    echo 137
+    echo 0
```

Comment:
Is this a temporary change, so that we can rebuild all images for the purpose of testing this PR?

Reply:
Right, that commit will be dropped on merge. The change was to be able to test all images, not only the ones which were modified 👍